flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/RemoteInputChannel.java

```diff
@@ -419,6 +442,17 @@ public int getAndResetUnannouncedCredit() {
 		return unannouncedCredit.getAndSet(0);
 	}
 
+	/**
+	 * Unblocks this channel from exactly once checkpoint.
+	 */
+	public void unblockChannel() {
```

Comment:
This can be removed if removing states by https://github.com/apache/flink/pull/11351#discussion_r397610517